app-directory/i18n/pages/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Layout, Text, Page, Code, Link } from '@vercel/examples-ui'
```

Comment:
It probably can be removed. In addition, we can comment that this will never be shown to the user due to the main redirect.

Reply:
I forgot about this ones